MaybeGame/GameMaster.cs

```diff
@@ -1,10 +1,63 @@
-﻿namespace MaybeGame
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MaybeGame
 {
-    partial class Program
+    partial class MainProgram
     {
         public class GameMaster
         {
+            private List<Player> _players { get; } = new();
+
+            public void PlayerDataDump()
```

Comment:
This isn't an entity which contains player data, this is a method. Better method name: **DumpPlayersData**

Reply:
That is correct